Makefile

```diff
@@ -13,12 +13,12 @@ GOPATH_DEFAULT := $(PWD)/.go
 export GOPATH ?= $(GOPATH_DEFAULT)
 TESTARGS_DEFAULT := "-v"
 export TESTARGS ?= $(TESTARGS_DEFAULT)
-PKG := $(shell awk  '/^package: / { print $$2 }' glide.yaml)
+PKG := $(shell awk  -F "\"" '/^ignored = / { print $$2 }' Gopkg.toml)
 DEST := $(GOPATH)/src/$(GIT_HOST)/$(BASE_DIR)
 DEST := $(GOPATH)/src/$(PKG)
 SOURCES := $(shell find $(DEST) -name '*.go')
 HAS_MERCURIAL := $(shell command -v hg;)
-HAS_GLIDE := $(shell command -v glide;)
+HAS_DEP := $(shell command -v dep;)
```

Comment:
`dep -v` isn't actually valid, although it will call dep and tell you if it's installed or not, so it works for the purpose here.

Reply:
The idea here is not to use `dep -v` but `command -v dep` (similar to how it was before with glide).
If dep is installed `command -v dep` will return the absolute path to dep. If it isn't installed it will return nothing.

Reply:
Yep, my bad; thanks for pointing it out